boards/st/nucleo_h753zi/nucleo_h753zi.dts

```diff
@@ -261,13 +261,13 @@ zephyr_udc0: &usbotg_fs {
 		/* backup slot: 256KB */
 		slot1_partition: partition@60000 {
 			label = "image-1";
-			reg = <0x00060000 DT_SIZE_K(256)>;
+			reg = <0x00060000 DT_SIZE_K(384)>;
 		};
 
 		/* storage: 256KB for settings */
-		storage_partition: partition@a0000 {
+		storage_partition: partition@c0000 {
 			label = "storage";
-			reg = <0x000a0000 DT_SIZE_K(256)>;
+			reg = <0x000c0000 DT_SIZE_K(256)>;
```

Comment:
Why not using 3 sectors even if it spreads into bank2 1st sector?
Even, why not using the overall 2MB flash, increasing slot0/slot1 sizes by several 128kB sectors?